macaroonbakery/third_party.py

```diff
@@ -55,3 +55,14 @@ def __eq__(self, other):
             self.version == other.version and
             self.ns == other.ns
         )
+
+
+class ThirdPartyInfo:
```

Comment:
Shouldn't this inherit from a namedtuple class?

Reply:
done